src/brogue/RogueMain.c

```diff
@@ -1171,12 +1174,28 @@ void gameOver(char *killedBy, boolean useCustomPhrasing) {
         displayMoreSign();
     }
 
-    if (!rogue.playbackMode) {
+    if (rogue.serverMode) {
+        blackOutScreen();
+        saveRecordingNoPrompt(recordingFilename);
+    }
+    else {
         if (saveHighScore(theEntry)) {
             printHighScores(true);
         }
         blackOutScreen();
-        saveRecording();
+        saveRecording(recordingFilename);
+    }
+
+    if(!rogue.playbackMode) {
+        if(!rogue.quit) {
+            notifyEvent(GAMEOVER_DEATH, theEntry.score, 0, theEntry.description, recordingFilename);
+        }
+        else {
+            notifyEvent(GAMEOVER_QUIT, theEntry.score, 0, theEntry.description, recordingFilename);
+        }
+    }
```

Comment:
Be careful about this formatting - most C code, including Brogue's, has a space between `if` and `(` and puts `else {` on the same line as the preceding `}`.

Reply:
I'm guessing you ran the code through a formatter originally for this repo - it would be good to find a checkstyle-like program for the CI pipeline.